chained-actions/index.json

```diff
@@ -0,0 +1,64 @@
+{
+    "pathwayTitle": "Apache OpenWhisk on OpenShift",
+    "icon": "fa-openshift",
+    "title": "Chained Actions",
+    "description": "Chaining Actions on OpenWhisk",
+    "difficulty": "intermediate",
+    "time": "15 minutes",
+    "details": {
+        "intro": {
+            "text": "00-intro.md",
```

Comment:
You named the file "intro.md" not "00-intro.md"

We really need to decide on whether we want to do the 00, 01, ....99 prefix or not so that we are consistent among all our scenarios.  I think I am in favor of using the prefix.  This seems to be a standard that others are using and I think it does make it easier to understand the ordering of steps just by looking at the directory rather than having to look at the index.json.

Reply:
Yeah, i've gone back and forth on it but I think you're right.  it's a little more annoying when reordering but I did that *once* and never since.  I'll redo it that way.